frontend/src/layout/onboarding/tour.js

```diff
@@ -0,0 +1,193 @@
+import React from 'react'
+import Tour from 'reactour'
+import { useValues, useActions } from 'kea'
+import { onboardingLogic, TourType } from './onboardingLogic'
+import { Button } from 'antd'
+
+export function PGTour() {
+    const { tourActive, tourType, tourStep } = useValues(onboardingLogic)
+    const { setTourFinish, setTourStep, updateOnboardingStep } = useActions(onboardingLogic)
+
+    function updateTourStatus(type, step) {
+        const tourSteps = determineTour(type)
+        if (step === tourSteps.length - 1) {
```

Comment:
Having trouble parsing how steps relate to tour type and why it's changing the onboarding steps. Is it for different flows or something?